nix/pkgs/default.nix

```diff
@@ -99,7 +99,10 @@ let
 
   # not available in 20.03 and we depend on several recent changes
   # including stylish-haskell support
-  nix-pre-commit-hooks = import (sources."pre-commit-hooks.nix");
+  nix-pre-commit-hooks = (pkgs.callPackage ((sources."pre-commit-hooks.nix") + "/nix/default.nix") {
+    system = builtins.currentSystem;
```

Comment:
Hmmmmmmmm. Is this right? I'm not sure this is right. What if I've manually passed in a different `system` argument? Then this is going to do the wrong thing. We should have `system` around somewhere, surely?

Reply:
As a matter of fact we do - it's passed to this very file. I'll just inherit it here.